src/dscanner/stats.d

```diff
@@ -132,14 +120,17 @@ void printTokenDump(string fileName)
 
 unittest
 {
-	import std.file;                                                                                                                                                                                                                                                                          
-	import core.stdc.stdio : freopen, stdout;                                                                                                                                                                                                                                                 
-	import core.sys.posix.unistd : dup, dup2;
 	import std.stdio : File;
-	import std.file : exists, remove;
+	import std.file : exists, remove, readText;
+	import dscanner.utils;
 
 	auto deleteme = "test.txt";
 	File file = File(deleteme, "w");
+	scope (exit)
+	{
+		assert(exists(deleteme));
+        remove(deleteme);
+	}
 
 	file.write(
```

Comment:
Why are we creating a file to put the text into it and then read that text back and pass it to the printTokenDump? We want to test only the functionality that we've added, which is the token dump. We can trust that the library implementation of read and write works correctly.